### DISCUSSION TITLE
More first issues

### DISCUSSION THREAD

**Discussion by Abitamim** [fastapi/fastapi#5499] [Questions] [answered]
There are only three issues with the label "good first issue". They are all years old and already solved. Would someone experienced be able to go in and tag a few dozen more issues with this tag for newer want-to-be-contributers like me?

**Comment by tiangolo** [comment, marked as answer]
Sorry, it's a lot of work to manage issues, and finding good first issues is also not easy. In many cases it's a lot easier to solve things directly. Also in many cases it's not possible to know if an issue will be hard or easy to solve until working on it.

But if you want to help, answering questions from others in issues is what helps the most, because that's what takes most of the time.
